charts/adoption-web/values.preview.template.yaml

```diff
@@ -13,7 +13,7 @@ nodejs:
   environment:
     APPINSIGHTS_KEY: '00000000-0000-0000-0000-000000000000'
     PCQ_ENABLED: 'true'
-    CCD_URL: 'http://ccd-data-store-api-adoption-cos-api-pr-249.service.core-compute-preview.internal'
+    CCD_URL: 'http://adoption-cos-api-pr-249-ccd-data-store-api'
```

Comment:
Remove this file chane, sync with R2master

Reply:
these changes are from me syncing with r2master